tests/translate/pythonir.py

```diff
@@ -0,0 +1,57 @@
+# TODO: I think I'll need this BaseNode class but not sure yet
+class BaseNode:
+    pass
+
+
+class ImportedSymbol:
+    def __init__(self, name, alias):
+        self.name = name
+        self.alias = alias
+
+    def __str__(self):
+        if self.alias:
+            return f"{self.name} as {self.alias}"
+        else:
+            return self.name
+
+    def __repr__(self):
+        return self.__str__()
+
+
+class Import:
+    def __init__(self, module, symbols):
+        self.module = module
+        self.symbols = symbols
```

Comment:
These two classes don't have a base class or child classes so I'm not sure why they exist. If they are data classes I suggest using that instead.

Reply:
Oh I've never heard of a dataclass, thanks for the tip.